content/docs/context.md

```diff
@@ -91,14 +91,22 @@ A more complex example with dynamic values for the theme:
 
 ## Consuming Multiple Contexts
 
+To keep context re-rendering fast, React needs to make each context consumer a separate node in the tree. 
+
 `embed:context/multiple-contexts.js`
 
+If two or more context values are often used together, you might want to consider creating your own render prop component that provides both. [React Context Composer](https://github.com/FormidableLabs/react-context-composer/) is a simple library to help with this.
```

Comment:
I would prefer not to recommend that library. It's fine, but it might not align with our future strategy for composing features. Also wary of recommending any one library over another one in the official docs. If we're going to link to any, we should link to more than one.

Also, let's not refer to anything as "simple." A pet peeve of mine :D

Reply:
Are you okay with leaving just the first sentence?

Reply:
Or maybe with a custom example that does that (without a library).

Reply:
Yeah that part is fine